app/models/work.rb

```diff
@@ -70,7 +71,7 @@ def self.display(type)
     end
 
     def self.options_for_select_box
-      all
+      (all - ['unspecified'])
```

Comment:
I know we already talked through this ad nauseam, and came to an agreement to do this, and no I'm not having second thoughts. _But_ using strings like this is the reason I do like the approach of using constants for these, even though _yes_ it's double the work. At least this usage of the string is inside the class. Just making a note of it, I don't think we need to change anything

Reply:
Yeah, I agree. I went back-and-forth on this a lot. I wonder if work type could extracted out into its own class, `WorkType` and managed that way. I _do_ like the enum approach with strings. Anyway, I'm open to refactoring this.

Reply:
I think it's fine as long as we're only using the string version inside the Types class/module. If they start leaking outside (which the enum should prevent), I think that's when we've got cause for concern

Reply:
So, I did notice that it's "leaking" into the PublishNewWork service, so I'm pushing a small change that identifies 'unspecified' as `Work::Types.unspecified` just like we do with `Work::Types.default`.